cxx/clean_relation.hh

```diff
@@ -115,29 +116,42 @@ class CleanRelation : public Relation<T> {
     return value;
   }
 
-  void unincorporate(const T_items& items) {
-    assert(data.contains(items));
-    ValueType value = data.at(items);
-    std::vector<int> z = get_cluster_assignment(items);
-    clusters.at(z)->unincorporate(value);
-    if (clusters.at(z)->N == 0) {
-      delete clusters.at(z);
-      clusters.erase(z);
-    }
+  void cleanup_data(const T_items& items) {
```

Comment:
Are you planning on adding tests for these new methods?

Reply:
Not sure I'll have time but I added TODOs.